packages/api/test/e2e/doughnut.e2e.ts

```diff
@@ -13,8 +13,6 @@
 // limitations under the License.
 
 import { hexToU8a } from '@polkadot/util';
```

Comment:
I don't think this needs to change, tests shouldn't break package build on normal install?

Reply:
ya... will revert this file